float64 x
float64 y
float64 z
float64 roll
float64 pitch
float64 yaw
float64 front
float64 back
float64 up
float64 left
float64 right
float64 zrange
float64 ref_x
float64 ref_y
float64 ref_z
float64 ref_roll
float64 ref_pitch
float64 ref_yaw
float64 front_x
float64 front_y
float64 front_z
float64 back_x
float64 back_y
float64 back_z
float64 left_x
float64 left_y
float64 left_z
float64 right_x
float64 right_y
float64 right_z

